File: src/main/java/nidhal/stackoverflowstatus/dto/StackExchangeResponse.java
package nidhal.stackoverflowstatus.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.Getter;
import lombok.Setter;
import nidhal.stackoverflowstatus.entity.Question;

import java.util.List;

@JsonIgnoreProperties(ignoreUnknown = true)
@Getter
@Setter
public class StackExchangeResponse {

    private List<Question> items;
    public boolean has_more;
    public int quota_max;
    public int quota_remaining;
}

File: backend/dotnet/stackoverflow_statistics/Dto/StackExchangeResponse.cs
using Newtonsoft.Json;
using stackoverflow_statistics.Models;

namespace stackoverflow_statistics.Dto

{
    [JsonObject(ItemNullValueHandling = NullValueHandling.Ignore)]
    public class StackExchangeResponse
    {
        [JsonProperty("items")]
        public List<Question> Items { get; set; }

        [JsonProperty("has_more")]
        public bool HasMore { get; set; }

        [JsonProperty("quota_max")]
        public int QuotaMax { get; set; }

        [JsonProperty("quota_remaining")]
        public int QuotaRemaining { get; set; }
    }
}